src/main/java/net/rptools/maptool/client/tool/PointerTool.java

```diff
@@ -296,11 +296,11 @@ public void exposeFoW(ZonePoint p) {
       }
 
       // Lee: fog exposure according to reveal type
-      if (zone.getWaypointExposureToggle()) FogUtil.exposeVisibleArea(renderer, exposeSet, false);
-      else {
+      if (zone.getWaypointExposureToggle()) {
+      } else {
```

Comment:
For the change in PointerTool I prefer changing the if into the negative rather than an empty if block